product_docs/docs/biganimal/release/using_cluster/tagging/create_and_manage_tags.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Creating and managing tags
+---
+
+BigAnimal supports tagging of the following resources:
+
+- Project
+- Cluster
+
+## Create and assign a tag
+
+You can create a tag using either of these methods:
+
+- [Create a tag and assign it to a resource]() 
+- [Create and assign a tag while creating a resource]()
+
+### Create a tag and assign it to a resource
+
+To create a tag:
+
+1. Log in to the BigAnimal portal.
+
+1. From the menu under your name in the top right of the panel, select **Tags**.
+
+1. From the Create Tag page, select **Create Tag**.
+
+1. Enter the tag name.
+
+1. Select the color for the tag.
+
+1. View the tag in **Preview**.
+
+1. Select **Save**.
+
+The generated tag is available for the user.
+
+To assign a tag to an existing cluster:
+
+1. Go to cluster's home page.
+
+1. In the clusters list, select the edit icon next to the cluster.
+
+1. On the Edit Cluster page, go to the **Cluster Settings** tab.
+
+1. 1. Under **Tags**, select **+**.
+
+1. In the search bar, enter the name of the tag and select the tag.
+
+1. To assign the tag, select **Save**.
+
+To assign a tag to an existing project:
+
+1. Go to the project's home page.
+
+1. In the projects list, select the edit icon next to the project.
+
+1. On the Edit Project page, under **Tags**, select **+**.
+
+1. In the search bar, enter the tag name and select the tag.
+
+1. To assign the tag, select **Save**.
```

Comment:
silly question, why do we have:
 1.Go to..
 1. ...
 1. ... 
 ???

Reply:
According to the style guide, we give numbers to indicate whether the steps are a procedure. 
 
To have actual sequence numbers in deployed docs we write in markdown like:
1. Bullet 1
1. Bullet 2

Reply:
but I see
![image](https://github.com/EnterpriseDB/docs/assets/18009508/7685fe85-1ffb-41b4-a3c4-5d19dab48acb)

Reply:
the previous comment is rendering 1 . 2. even when I write 1. 1.  :shrug:

Reply:
yes, that's how it works in deployment too..
![Uploading Screenshot 2024-03-14 at 1.55.08 PM.png…]()

Reply:
It's better practice to use this encoding in case you add steps later. You don't have to renumber. Markdown has very little in the way of semantic coding capabilities, but this is one feature I recommend as best practice.